testkit/src/test/scala/org/apache/pekko/testkit/PekkoSpec.scala

```diff
@@ -59,6 +60,15 @@ object PekkoSpec {
     ConfigFactory.parseMap(map.asJava)
   }
 
+  def resourcePath(resourceName: String): String = {
```

Comment:
could you add comments on these to say they are designed to help running tests on Windows?